material-ui/material-ui.scss

```diff
@@ -120,3 +120,106 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  z-index: 0;
+  position: relative;
+  > {
+    input {
+      appearance: none;
+      -moz-appearance: none;
+      -webkit-appearance: none;
+      z-index: -1;
+      position: absolute;
+      left: -10px;
+      top: -8px;
+      display: block;
+      margin: 0;
+      border-radius: 50%;
+      width: 40px;
+      height: 40px;
+      background-color: $material-ui-blue;
+      box-shadow: none;
+      outline: none;
+      opacity: 0;
+      transform: scale(1);
+      pointer-events: none;
+      transition: opacity 0.3s, transform 0.2s;
+    }
+    span {
+      display: contents;
+      width: 100%;
+      cursor: pointer;
+      &::before {
+        content: "";
+        display: inline-block;
+        box-sizing: border-box;
+        margin: 3px 11px 3px 1px;
+        border: solid 2px;
+        /* Safari */
```

Comment:
What is this comment about? Does it mean that the styles below are specific for Safari only?

Reply:
Yes I had safari specific there before, let me clean that up